pkg/interface/publish/src/js/components/skeleton.js

```diff
@@ -30,7 +30,7 @@ export class Skeleton extends Component {
             active={props.active}
             notebooks={props.notebooks}
             />
-          <div className={"h-100 w-100 " + rightPanelHide} style={{
+          <div className={"h-100 w-100 overflow-container " + rightPanelHide} style={{
```

Comment:
I'd probably put this in its own `<div>` as the parent div of everything in `note.js` — we're going to have a navigation bar to collapse the sidebar and navigate back to the Notebook view, and putting the overflow class here will let the user scroll the entire right panel as one container. But we'll want the nav bar to stay at the top. (Which means note.js will probably have yet another parent `<div>`, followed by the nav bar component, followed by the `overflow-container` div, followed by the `flex justify-center` one.

Reply:
I tried doing that just now, and it's not working. I think it might have something to do with h-100?

Reply:
The overflow-container parent div also needs `h-100`. This works:

<img width="407" alt="image" src="https://user-images.githubusercontent.com/20846414/73326416-17913380-4220-11ea-9638-acc14c991082.png">

Reply:
Added this and it works. Thanks.